test/shared/src/main/scala/zio/test/mock/Method.scala

```diff
@@ -16,7 +16,34 @@
 
 package zio.test.mock
 
-trait Method[+A, -B] {
+import com.github.ghik.silencer.silent
+import zio.test.Assertion
+
+/**
+ * A `Model[M, I, A]` represents a capability of module `M` that takes an
+ * input `I` and returns an effect that may produce a single `A`.
+ */
+trait Method[M, I, A] { self =>
+
+  import Method._
+
+  /**
+   * Provides the `Assertion` on method arguments `I` to produce `ArgumentExpectation`.
+   *
+   * Available only for methods that do take arguments.
+   */
+  @silent("parameter value ev in method apply is never used")
```

Comment:
Instead of using `@silent` you can also do `val _ = ev` in the body of the method. I'm not sure which is better here.

Reply:
Since the silencer was already in the dependencies I leaned towards using it. But I can change that if you guys prefer other solution.

Reply:
Fine with me.